monika.example.yml

```diff
@@ -0,0 +1,94 @@
+probes:
+  - id: '1'
```

Comment:
Rather than one giant probe example with sprinkled comments, it's better to follow the style of the [docs](https://monika.hyperjump.tech/guides/examples): a few examples, starting with the minimal one and becoming more complex, where new config keys (`method`, `timeout`, etc) are described with comments.